src/pks/transport/transport_ats.hh

```diff
@@ -457,7 +457,7 @@ protected:
   Key molar_density_key_;
   Key solid_residue_mass_key_;
   Key water_content_key_;
-  Key water_src_key_;
+  Key water_src_key_, trans_srs_key_;
```

Comment:
Can we come up with a better name than trans_srs?  I have no idea what that means.

Reply:
Changed to solute_src_key_. I hope it is more explanatory.
Although it is still a bit tricky. This is the design for defining source functions via fields.